src/main/java/org/skriptlang/skript/bukkit/itemcomponents/ItemComponentModule.java

```diff
@@ -0,0 +1,60 @@
+package org.skriptlang.skript.bukkit.itemcomponents;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.ClassInfo;
+import ch.njol.skript.classes.Parser;
+import ch.njol.skript.lang.ParseContext;
+import ch.njol.skript.registrations.Classes;
+import org.skriptlang.skript.addon.AddonModule;
+import org.skriptlang.skript.addon.SkriptAddon;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableModule;
+
+import java.io.IOException;
+
+public class ItemComponentModule implements AddonModule {
+
+	@Override
+	public boolean canLoad(SkriptAddon addon) {
+		return Skript.isRunningMinecraft(1, 21, 2);
```

Comment:
Is there a Paper class we could check for the existence of? Usually a bit more reliable